content/components/recipes/details/ExperimenterDetails.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+
+import CollapsibleSection from "devtools/components/recipes/details/CollapsibleSection";
+import { useExperimenterDetailsData } from "devtools/contexts/experimenterDetails";
+
+const ExperimenterDetails: React.FunctionComponent = () => {
+  const data = useExperimenterDetailsData();
+  return (
+    <CollapsibleSection
+      headerButtons={<></>}
+      title={
+        <>
+          <h6>Experimenter Details</h6>
+        </>
+      }
```

Comment:
The fragment (`<></>`) here isn't needed. That is only needed if you need a JSX element that has `n != 1` children.